frontend/routes/teamspaces/components/modelItem/modelItem.component.tsx

```diff
@@ -139,7 +139,7 @@ export class ModelItem extends React.PureComponent<IProps, IState> {
 
 			if (!isHidden && !isPendingStatus(this.props.status)) {
 				return renderWhenTrue((
-					<Tooltip title={label} key={index}>
+					<Tooltip enterDelay={50} title={label} key={index}>
```

Comment:
I added this delay (which is not noticeable for an ordinary mortal) because the tooltip sometimes is getting the wrong position. It's because tooltip is setting the position before the menu's animation finished